integrations/symfony-bundle.rst

```diff
@@ -167,6 +147,39 @@ Authentication
                 plugins: ['httplug.plugin.authentication.my_wsse']
 
 
+List of services
+````````````````
+
++----------------------------------+---------------------------------------------------------------------+
+| Service id                       | Description                                                         |
++==================================+=====================================================================+
+| httplug.message_factory          | Service* that provides the `Http\Message\MessageFactory`            |
++----------------------------------+---------------------------------------------------------------------+
+| httplug.uri_factory              | Service* that provides the `Http\Message\UriFactory`                |
++----------------------------------+---------------------------------------------------------------------+
+| httplug.stream_factory           | Service* that provides the `Http\Message\StreamFactory`             |
++----------------------------------+---------------------------------------------------------------------+
+| httplug.client.[name]            | | This is your HttpClient that you have configured.                 |
+|                                  | | With the configuration below the name would be `acme_client`.     |
++----------------------------------+---------------------------------------------------------------------+
+| httplug.client                   | This is the first client configured or a client named `default`.    |
++----------------------------------+---------------------------------------------------------------------+
+| | httplug.plugin.content_length  | | These are plugins that are enabled by default.                    |
+| | httplug.plugin.decoder         | | These services are not public and may only be used when configure |
+| | httplug.plugin.error           | | HttpClients or other services.                                    |
+| | httplug.plugin.logger          |                                                                     |
+| | httplug.plugin.redirect        |                                                                     |
+| | httplug.plugin.retry           |                                                                     |
+| | httplug.plugin.stopwatch       |                                                                     |
++----------------------------------+---------------------------------------------------------------------+
+| | httplug.plugin.cache           | | These are plugins that are disabled by default.                   |
+| | httplug.plugin.cookie          | | They need to be configured before they can be used.               |
+| | httplug.plugin.history         | | These services are not public and may only be used when configure |
+|                                  | | HttpClients or other services.                                    |
```

Comment:
s/HttpClients/clients/

same above - and same question about phrasing as above.